public/app/plugins/datasource/loki/components/LokiQueryEditor.tsx

```diff
@@ -38,9 +38,13 @@ export const LokiQueryEditor = React.memo<LokiQueryEditorProps>((props) => {
   const { flag: explain, setFlag: setExplain } = useFlag(lokiQueryEditorExplainKey);
 
   const timerange = datasource.getTimeRange();
+  const predefinedOperations = datasource.predefinedOperations;
   const previousTimerange = usePrevious(timerange);
 
   const query = getQueryWithDefaults(props.query);
+  if (config.featureToggles.lokiPredefinedOperations && !query.expr) {
```

Comment:
```suggestion
  if (config.featureToggles.lokiPredefinedOperations && !query.expr && predefinedOperations) {
```

to avoid cases were we might end up with `null`/`undefined`/`empty string` values.